### DISCUSSION TITLE
Browse by resource type

### DISCUSSION THREAD

**Discussion by fenekku** [inveniosoftware/product-rdm#193] [Ideas]
This might be solved by having resource types be a vocabulary and working like languages and others. BUT browsing for resource type is still very much wanted.

**Comment by lnielsen** [comment]
Regarding the browse view, is it something like this?

![Screenshot 2021-03-17 at 15 54 09](https://user-images.githubusercontent.com/1698163/111487889-03b4d380-8739-11eb-8dfa-9e56a167d35c.png)

**Comment by fenekku** [comment]
Ah sorry, the image would be something else (and yes they would probably like this, but they didn't ask for it yet).

What I meant was ressource types lists all resource type but like language a resource type can be honed in on by typing.

[a comment by an automated account is not shown]

**Comment by lnielsen** [comment]
I've moved to the product board instead. For advanced search capabilities and e.g. inline search auto-completion the issues is #35. This issue is specifically about making a browsable collection.

**Comment by lnielsen** [comment]
Related to #191

**Comment by ntarocco** [comment]
This will be solved by https://github.com/inveniosoftware/product-rdm/discussions/222
